docs/modules/ROOT/partials/install/bootstrap-nodes.adoc

```diff
@@ -98,6 +98,32 @@ terraform apply
 
 ifeval::["{provider}" == "cloudscale"]
 . Add the DNS records for etcd shown in output variable `dns_entries` from the previous step to the cluster's parent zone
+
+. Apply the manifests for the cloudscale machine-api provider
++
+[source,bash,subs="attributes+"]
+----
+cat ../machine-api-provider-cloudscale/00_secrets.yaml  | yq '(select(document_index==0) | .stringData.token) = "'$(vault kv get -format=json clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cloudscale/ | jq -r .data.data.token)'" ' | kubectl apply -f -
```

Comment:
This could be the following:

```suggestion
kapitan refs --reveal --refs-path ../../refs ../machine-api-provider-cloudscale/00_secrets.yaml | kubectl apply -f -
```

Reply:
Oh, TIL